templates/timeseries.html

```diff
@@ -10,43 +10,44 @@
         <div class="row row-eq-height">
           <div id="graph-container" class="col-md-9 col-lg-10"></div>
           <div class="col-md-3 col-lg-2">
-		  	<span>Add staff</span>
-			<div>
-				<input id="addStaff" class="selector left" style="background-color:#c9fdc9" type="text">
-				<button onClick="addChart()" class="ui-button ui-corner-all ui-widget right"></button>
-			</div>
-            <span>Remove staff</span>
-			<div>
-				<input id="removeStaff" class="selector left" style="background-color:#ffb2b2" type="text">
-				<button onClick="removeChart()" class="ui-button ui-corner-all ui-widget right"></button>
-			</div>
+      	    <div class="form-group has-feedback" id="addStaff-form-group">	    
+              <input id="addStaff" class="form-control selector left" type="text" style="border-width: 5px;" placeholder="Staff Username"></input>
+            </div>
+      	    <ul id="userEntries" class="list-group"></ul>
           </div>
         </div>
       </div>
     </body>
-	<style>
-	.right {
-		height: 27px;
-		width: 20%;
-		vertical-align: middle;
-	}
-	.left {
-		width: 75%;
-		vertical-align: middle;
-	}
-	</style>
-
+    <style>
+      .right {
+  	height: 27px;
+  	width: 20%;
+        vertical-align: middle;
+      }
+      .left {
+  	width: 75%;
+        vertical-align: middle;
+      }
```

Comment:
Looks like the indentation got a little messed up here.

Reply:
Indentation seems to be displaying inconsistently on vim vs github.
![vimvsgitinconsistency](https://user-images.githubusercontent.com/16551944/33631688-25991d0a-d9c0-11e7-8947-d66d0e9ef8ec.jpg)

Reply:
Sounds good to me, I'll merge it in.